ReactiveTask/Task.swift

```diff
@@ -391,14 +428,19 @@ extension Signal where Value: TaskEventType {
 	}
 }
 
-/// Launches a new shell task, using the parameters from `taskDescription`.
+/// Launches a new shell task.
+///
+/// - Parameters:
```

Comment:
I just discovered this yesterday [in the apple/swift documentation](https://github.com/apple/swift/blob/master/docs/DocumentationComments.md). :smile: